math/TestWaveConversion.cpp

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) 2020 Nicholas Corgan
+// SPDX-License-Identifier: BSL-1.0
+
+#include <Pothos/Testing.hpp>
+#include <Pothos/Framework.hpp>
+
+#include <complex>
+#include <cstring>
+#include <iostream>
+
+//
+// Common test code
+//
+
+// Test two arrays for near-equality
+#if !defined(POTHOS_TEST_CLOSEA)
```

Comment:
looks useful, could I get a pull for this in Testing.hpp?

dont worry about compatibility, just ifdef POTHOS_TEST_CLOSEA for the tests below

Reply:
I have a branch somewhere local with a few of these. I'll push up the branch when I find it.